src/nyc_trees/apps/event/templates/event/event_list_page.html

```diff
@@ -39,8 +39,12 @@
                 {% include "event/partials/event_list.html" with event_list=all_events %}
             {% endif %}
           </div>
-          <div class="tab-pane" id="map">
+          <div class="tab-pane" id="map-tab" style="position: relative; width: 100%; height: 440px;">
+            <div id="map" style="width: 100%; height: 400px;">
+            </div>
+            <div id="action-bar" style="position: absolute; width: 100%; height: 40px;"></div>
```

Comment:
@jfrankl I'm not a huge fan of inline styling, but we haven't really styled any of our map stuff yet and I didn't want to trip you up.

Should I leave these in here, or pull them out to SCSS?